biz.aQute.resolve/test.bndrun

```diff
@@ -0,0 +1,13 @@
+-standalone: true
```

Comment:
Since this is a file for testing, it should not live in the root of the project but inside the `testdata` folder.

Reply:
Yeah, makes sense. Will fix it.

Reply:
DONE ✔️